activerecord/lib/active_record/normalization.rb

```diff
@@ -51,7 +51,8 @@ module ClassMethods
       #
       # ==== Options
       #
-      # * +:with+ - The normalization to apply.
+      # * +:with+ - Any callable object that accepts the attribute's value as
+      #   its sole argument, and returns it normalized.
```

Comment:
I think this is a good clarification.  Do you think it would be too verbose to also retain "The normalization to apply"?  Such as, "The normalization to apply. Any callable object that...".  In the paragraphs above, I make several references to "the normalization" and "applying", trying to establish terminology.  Likewise, the `:apply_to_nil` option includes "Whether to apply the normalization...".

Reply:
I think that would be fine.

I did not include it because, to me, the value is the _normalizer_. The normalization is what the normalizer does _later_, when invoked.

For example:

```ruby
normalizes :name, with: UnicodeNFCNormalizer.new
```

By saying "returns [the value] normalized", the text entails this without giving it a name.

However, if you like that original sentence that would be fine with me too, your call :).

Reply:
"Normalizer" is also a good term.  `NormalizedValueType` actually uses it instead.  I stuck with "normalization" in the other paragraphs just to limit the number of terms I introduced.

Let's go with what you have now. :+1: